modules/nf-scil/betcrop/fslbetcrop/main.nf

```diff
@@ -23,31 +23,53 @@ process BETCROP_FSLBETCROP {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     def b0_thr = task.ext.b0_thr ? "--b0_thr " + task.ext.b0_thr : ""
-    def bet_dwi_f = task.ext.bet_dwi_f ? "-f " + task.ext.bet_dwi_f : ""
+    def bet_f = task.ext.bet_f ? "-f " + task.ext.bet_f : ""
 
     """
     export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
     export OMP_NUM_THREADS=1
     export OPENBLAS_NUM_THREADS=1
 
-    scil_extract_b0.py $dwi $bval $bvec ${prefix}__b0.nii.gz --mean \
-        $b0_thr --force_b0_threshold
-    bet ${prefix}__b0.nii.gz ${prefix}__b0_bet.nii.gz -m -R $bet_dwi_f
-    scil_image_math.py convert ${prefix}__b0_bet_mask.nii.gz ${prefix}__b0_bet_mask.nii.gz --data_type uint8 -f
-    mrcalc $dwi ${prefix}__b0_bet_mask.nii.gz -mult ${prefix}__dwi_bet.nii.gz -quiet -nthreads 1
+    if [ -v "$bval" ]
```

Comment:
I'm having a hard time finding documentation on the `-v`check in bash. I think you got this from stackoverflow or something, am I right ? Can you link me to something ?

Reply:
I found the command here : https://unix.stackexchange.com/questions/212183/how-do-i-check-if-a-variable-exists-in-an-if-statement